benthomasson/eda/plugins/modules/upcase.py

```diff
@@ -1,17 +1,32 @@
 #!/usr/bin/python
 
+# This file is part of Ansible
```

Comment:
Is this true?  It isn't part of Ansible core.  Do we apply GPL to all our files?

Reply:
This is the only file complained missing license information. I copied it from other files, may need to be updated for our purpose

Reply:
Please change the license to Apache v2

Reply:
@hsong-rh @benthomasson That is a test module I created. So that we can test it

Reply:
@mkanoor The line `# This file is part of Ansible` is removed